build.cake

```diff
@@ -25,7 +25,7 @@ PackageDefinition NUnitConsoleRunnerNet80Package;
 PackageDefinition NUnitEnginePackage;
 PackageDefinition NUnitEngineApiPackage;
 PackageDefinition NUnitConsoleRunnerChocolateyPackage;
-PackageDefinition NUnitConsoleZipPackage;
+//PackageDefinition NUnitConsoleZipPackage;
```

Comment:
Should this be removed?

Reply:
Yes. It was indefinite because JetBrains uses it in their installation process for TeamCity. But as of a few days ago, we know that we can drop it for V4. They will use the nuget packages.